deploy/crds/shipwright.io_buildstrategies.yaml

```diff
@@ -1561,7 +1561,8 @@ spec:
                       - driver
                       type: object
                     description:
-                      description: Description of the Build Volume
+                      description: "Description of the Build Volume \n NOTICE: Description
+                        is deprecated, and will be removed in a future release."
```

Comment:
Can you check this ? If I remember it correctly, we want to retain description fields in the strategy (like we have for parameters there) so that strategy authors can describe the purpose of the entity they define. The build users then don't have description fields in Build and BuildRun (we have no one in paramValues).

It might be that a common Go type is used here that we may have to split now between strategy spec and Build(Run) spec.

Reply:
thanks, good catch, let me add a new commit to address this.